master/src/ComputerZoomWidget.h

```diff
@@ -38,15 +38,26 @@ class ComputerZoomWidget : public QWidget
 	ComputerZoomWidget( const ComputerControlInterface::Pointer& computerControlInterface );
 	~ComputerZoomWidget() override;
 
+	VncViewWidget* vncView() const
+	{
+		return m_vncView;
+	}
+
 protected:
+	bool eventFilter( QObject* object, QEvent* event ) override;
+
 	void resizeEvent( QResizeEvent* event ) override;
+	void closeEvent( QCloseEvent* event ) override;
 
 private:
 	void updateSize();
 	void updateComputerZoomWidgetTitle();
 
-	void closeEvent( QCloseEvent* event ) override;
+	int currentScreen;
```

Comment:
Please prefix member variables with `m_`. Also you can initialize it directly via `int m_currentScreen{-1}` and save the initialization in the constructor.

Reply:
Okay. Done.